deploy/operator/apply/07_OperatorDeployment.yaml

```diff
@@ -24,6 +24,10 @@ spec:
             value: "TRUE"
           - name: WEBHOOK_TLS_UNSAFE_KEYSTORE_ENABLED
             value: "TRUE"
+          - name: VERSIONED_HOSTS_ENABLED
```

Comment:
Make sure these are remembered in the gateway cluster deployment repo

Reply:
In fact, that will auto kick off and fail... Not the best system we have with these files being maintained in two places... 😢